replicator-security/README.md

```diff
@@ -30,9 +40,16 @@ In all containers, minimal non-replicator configuration is supplied.
 
 The [test data](testData/testData.txt) is loaded into the source topic, and it can be modified to suit testing requirements.
 
-## Creating and verifying the environments
+## Setup and verifying the environments
+
+Before running the demos appropriate security resources must be created to create these run:
 
-In all environments no actions are taken on the destination brokers. Because of this we can verify the environment by consuming from the test topic on the destination cluster.
+```
+cd scripts/security
+./create_certs.sh
+```
+
+In all demos no actions are taken on the destination brokers. Because of this we can verify the environment by consuming from the test topic on the destination cluster.
```

Comment:
What does this mean?  "no actions are taken on the destination brokers"

Reply:
It meant that replicator was the only thing creating topics and writing to the destination but i agree it's confusing so have removed it.